app/mailers/devise_mailer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class DeviseMailer < Devise::Mailer
+  layout 'mailer'
+
+  def confirmation_instructions(record, token, opts = {})
+    @token = record.confirmation_token
+    opts[:subject] = 'Confirmation code'
+    opts[:from] = 'Otter Whispers'
```

Comment:
i am keeping this because, i couldnt find place to override that from. 
if there is no from then email look like from <otterwhispers@gmail.com>.
if there is from then it look from Otter Whisper
email: otterwhispers@gmail.com